rllib/algorithms/ppo/tests/test_ppo.py

```diff
@@ -331,7 +359,65 @@ def get_value(fw=fw, policy=policy, log_std_var=log_std_var):
             # Check the variable is initially zero.
             init_std = get_value()
             assert init_std == 0.0, init_std
-            batch = compute_gae_for_sample_batch(policy, FAKE_BATCH.copy())
+            batch = compute_gae_for_sample_batch(policy, CARTPOLE_FAKE_BATCH.copy())
+            if fw == "torch":
+                batch = policy._lazy_tensor_dict(batch)
+            policy.learn_on_batch(batch)
+
+            # Check the variable is updated.
+            post_std = get_value()
+            assert post_std != 0.0, post_std
+            trainer.stop()
+
+    def test_ppo_free_log_std_with_rl_modules(self):
```

Comment:
Can we make sure this test runs with RLModule and Learner enabled? trainable_variables is not callable anymore.